packages/deployment/scripts/integration-test.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash
+set -ueo pipefail
+
+real0=$(readlink "${BASH_SOURCE[0]}" || echo "${BASH_SOURCE[0]}")
+thisdir=$(cd "$(dirname -- "$real0")" > /dev/null && pwd -P)
+
+export NETWORK_NAME=${NETWORK_NAME-localtest}
+
+mkdir -p "$NETWORK_NAME/setup"
+cd "$NETWORK_NAME/setup"
+
+# Speed up the docker deployment by pre-mounting /usr/src/agoric-sdk.
+DOCKER_VOLUMES="$(cd "$thisdir/../../.." > /dev/null && pwd -P):/usr/src/agoric-sdk" \
+  "$thisdir/docker-deployment.cjs" > deployment.json
+
+# Set up the network from our above deployment.json.
+"$thisdir/setup.sh" init --noninteractive
+
+# Go ahead and bootstrap.
+exec "$thisdir/setup.sh" bootstrap ${1+"$@"}
```

Comment:
Is there a reason to exec here?

Should the test be nice and cleanly stop the bootstrapped chain?

Reply:
Those are reasonable improvements, but not necessary for the Github Actions test, so I didn't do them.